CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.24.0 (December 13, 2021)
+
+IMPROVEMENTS
+
+* Added Get, Update, DeleteKeys, Associate, Disassociate, DisassociateAll, and ListAssociations for blockstorage QoS v3 via [GH-2283](https://github.com/gophercloud/gophercloud/pull/2283) and [GH-2284](https://github.com/gophercloud/gophercloud/pull/2284)
+* CI: enable Neutron Dynamic Routing service [GH-2271](https://github.com/gophercloud/gophercloud/pull/2271)

```

Comment:
Does this really belong to the changelog? Looking at the historical content in the changelog, I'd say no.